### PR TITLE
SuccessFactors e2e tests

### PR DESCRIPTION
**Pull request by Vipinofficial11**

### CODE REVIEW

File: src/e2e-test/features/successfactors/DesignTime.feature [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    And Enter input plugin property: "referenceName" with value: "Referencename"
+    And Enter input plugin property: "baseURL" with value: "admin.baseurl" for Credentials and Authorization related fields
+    And Enter input plugin property: "entityName" with value: "<EntityName>"
+    And Enter input plugin property: "username" with value: "admin.username" for Credentials and Authorization related fields
+    And Enter input plugin property: "password" with value: "admin.password" for Credentials and Authorization related fields
+    And Enter textarea plugin property: "selectOption" with value: "<SelectedFields>"
+    And Click on the Validate button
+    Then Validate output schema with expectedSchema "<ExpectedSchema>"
+    Examples:
+      | EntityName      | SelectedFields                  | ExpectedSchema        |
```
Comment (itsankit-google) [inline src/e2e-test/features/successfactors/DesignTime.feature]: nit: indentation

File: src/e2e-test/java/io/cdap/plugin/tests/hooks/TestSetupHooks.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+  @After(order = 1, value = "@BQ_SINK_CLEANUP")
+  public void deleteTempTargetBQTable() throws IOException, InterruptedException {
+    String bqTargetTable = PluginPropertyUtils.pluginProp("bqtarget.table");
+    try {
+      BigQueryClient.dropBqQuery(bqTargetTable);
+      BeforeActions.scenario.write("BigQuery Target table: " + bqTargetTable + " is deleted successfully");
+      bqTargetTable = StringUtils.EMPTY;
+    } catch (BigQueryException e) {
+      if (e.getMessage().contains("Not found: Table")) {
```
Comment (itsankit-google) [inline src/e2e-test/java/io/cdap/plugin/tests/hooks/TestSetupHooks.java]: nit: use `e.getCode() == 404`

**Review by itsankit-google — APPROVED** [review]
lgtm

File: src/e2e-test/features/successfactors/DesignTime.feature (hunk 1 above)
Comment (Vipinofficial11) [reply src/e2e-test/features/successfactors/DesignTime.feature]: updated(6d9914b688d1fe1a2f422a990f61616f837654b3)

File: src/e2e-test/java/io/cdap/plugin/tests/hooks/TestSetupHooks.java (hunk 2 above)
Comment (Vipinofficial11) [reply src/e2e-test/java/io/cdap/plugin/tests/hooks/TestSetupHooks.java]: updated (https://github.com/data-integrations/successfactors-plugins/pull/13/commits/6d9914b688d1fe1a2f422a990f61616f837654b3#diff-6cafbcddecea707397feb9d1901df1948a5bd07a82e11d564307d139c3c95e33R73)

**Review by itsankit-google — COMMENTED** [review]
Please squash commits.

**Comment by Vipinofficial11** [comment]
> Please squash commits.

commits Squashed.

**Pull request merged**